app/controllers/application_controller.rb

```diff
@@ -1,5 +0,0 @@
-class ApplicationController < ActionController::Base
```

Comment:
Even if it's empty we still probably want the ApplicationController class as it's a rails convention.